tools/vipps-login-api.postman_collection.json

```diff
@@ -40,6 +40,11 @@
 			"request": {
 				"method": "GET",
 				"header": [
+					{
+						"key": "Merchant-Serial-Number",
+						"value": "{{merchantSerialNumber}}",
+						"type": "default"
+					},
```

Comment:
Have you tested that it works sending these? Or at least that nothing breaks?

Reply:
yes